src/api.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { formatDate } from "./utils/formatDate";
+
+const ncNewsApi = axios.create({
+    baseURL: 'https://val-northcoders.onrender.com/api',
+});
+
+export const getArticles = (query = '') => {
+    return ncNewsApi.get(`/articles${query}`)
+        .then((response) => {
+            return response.data.articles;
+        })
+        .catch((err) => {
```

Comment:
Rarther than catching the errors and logging them in the API it might be better to have a catch block in the components themselves so an error message can be displayed